src/coreclr/jit/jitconfigvalues.h

```diff
@@ -463,7 +463,10 @@ CONFIG_INTEGER(JitClassProfiling, W("JitClassProfiling"), 1)
 CONFIG_INTEGER(JitEdgeProfiling, W("JitEdgeProfiling"), 1)
 
 // Profile consumption options
-CONFIG_INTEGER(JitDisablePGO, W("JitDisablePGO"), 0) // Ignore pgo data
+CONFIG_INTEGER(JitDisablePgo, W("JitDisablePgo"), 0) // Ignore pgo data for all methods
```

Comment:
Would this be easier to understand in reverse? E.g., `JitEnablePgo=0`? Like we have `COMPlus_TieredCompilation=0`.

Doesn't matter to me, but sometimes negatives get confusing. We certainly aren't consistent here.

Reply:
Since PGO is on by default and this option is mainly to do top-level triage/workaround for PGO issues, I think "disable" is less confusing.

Though when I needed to binary search I found having an "enable" less confusing.... so am not self-consistent here either.